EIPS/eip-770.md

```diff
@@ -0,0 +1,124 @@
+## Preamble
+
+    EIP: 770 <or to be assigned by an editor>
+    Title: Decentralized Booking Protocol Standard
+    Author: Vidal Chriqui <vidal@btuprotocol.com>
+    Type: Standard
+    Category : ERC
+    Status: Draft
+    Created: 2017-12-26
+    Requires : ERC 20
+
+## Abstract
+
+The following describes standard functions for a decentralized booking protocol that can be used to reserve any kind of resource (Hotel, concert ticket, restaurant table).
+The protocol involves the following participants :
+
+- Provider : an entity providing its own resource or a third-party party resource to be booked
+- Booker : an entity that books the resource for itself or a third party
+- Booking infrastructure : a public "location" where providers can "post" resource availability data and where these resources can be searched by bookers. The booking infrastructure facilitates signaling between providers and bookers by aggregating a high volume of resource type and availabilities.
+- RES smart contract : A booking smart contract implementation of the decentralized standardized BTU protocol.
+
+## Motivation
+
+The Booking Token Unit (BTU) protocol is a building block for any decentralized application (dApp) or web site willing to implement booking features for their end-users.
+This standard also brings interoperability among decentralized applications that incorporate it.
+We strongly believe that a  transparent and public inventory enabled by an open-source standardized protocol would considerably lower the entry barriers to many Internet booking markets.
+
+## Specification
+
+The BTU protocol involves potentially ten steps and our standardized decentralized reservation contract (RES). Some of following steps can be relayed off-chain, but are always settled on-chain.
+
+<img src="./eip-770/protocol-steps.png">
```

Comment:
The image contains an Ethereum logo.  Ethereum logos are distributed under Creative Commons Attribution 3.0.  Please follow the term of the license.

Reply:
Thanks for this reminder that will be taken into account in next submission